src/adv.py

```diff
@@ -49,3 +157,55 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+while True:
+    curr_room = Player_one.current_room
+    print(room[curr_room])
+    user_input = input("What do you wish to do?(Move via N,S,W,E, or q)")
```

Comment:
Perhaps add more error handling and print them out so it is easier to read

Reply:
Will do